code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -169,6 +170,9 @@
 	scandisk()
 	point_at_target(the_disk)
 
+/obj/item/pinpointer/advpinpointer/AltClick(mob/user)
+	toggle_mode()
```

Comment:
can you check to make sure that observers and people out of range can't toggle mode on this?

Reply:
Observers cant use it but you can use it from range... pushing fix in next commit